osal/src/os/posix/src/os-impl-fpu.c

```diff
@@ -20,17 +20,14 @@
 
 /**
  * \file     os-impl-fpu.c
- * \ingroup  posix
- * \author   joseph.p.hickey@nasa.gov
+ * \ingroup  FreeRTOS
  *
  */
 
 /****************************************************************************************
                                     INCLUDE FILES
  ***************************************************************************************/
 
-#include "os-posix.h"
-#include "bsp-impl.h"
```

Comment:
Is it necessary to remove this include?

Reply:
the posix which was renamed should stay. The bsp-impl should be removed, rtems version at least does not include it.